src/lib/card/card.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+    selector: 'plex-card',
+    template: `
+    <div class="card" [ngClass]="{'open': !closed}">
+        <plex-button *ngIf="!closed" type="danger" size="sm" icon="close" (click)="toggleClose()"></plex-button>
+        <plex-button *ngIf="closed" type="info" size="sm" icon="help" (click)="toggleClose()"></plex-button>
+        <ng-container *ngIf="!closed">
+            <div class="card-header">
+                <plex-title titulo="{{ titulo }}" size="sm">
```

Comment:
```suggestion
                <plex-title [titulo]="titulo" size="sm">
```
Es más optimo porque bindea directamente la variable.

Reply:
Ah, es verdá! :+1:

Reply:
> 1. la lógica me indica que el nombre plex-card hace referencia a las card que van como items de un listado. No debería ser algo más tipo plex-tooltip, plex-help, plex-infogram, plex-luthor?
**Quedó como plex-help**
> 2. Siempre el disparador va a ser un botton?
**Si, quizá se extienda la funcionalidad del botón (circular, sin fondo, etc)**
> 3. Ya hay algún caso de uso?
**Si: reglas TOP, Información gral de una prestación, de una agenda, etc**
> 4. Al igual que el dropdown, se debería desaparecer al hacer click en cualquier lugar de la pantalla.
**No se pensó que funcione así, porque el ícono/texto se reemplaza por un botón para cerrar, queda en consideración para el siguiente ciclo, che**